config/twig_hooks/admin/media/update.yaml

```diff
@@ -47,11 +47,11 @@ sylius_twig_hooks:
                 priority: 0
 
         'sylius_cms.admin.media.update.content.form.sections.media':
-            file:
-                template: '@SyliusCmsPlugin/admin/media/form/sections/media/file.html.twig'
-                priority: 400
             type:
                 template: '@SyliusCmsPlugin/admin/media/form/sections/media/type.html.twig'
+                priority: 400
+            file:
+                template: '@SyliusCmsPlugin/admin/media/form/sections/media/file.html.twig'
```

Comment:
Why are you changing the order? Is it necessary? 🤔  Because changing the priority of the Twig Hooks affects the end application, so if it is not necessary, it would be good to leave it as it is now.

Reply:
I changed this to have the same order of fields during editing and creating a new “media"